dpctl/tensor/_elementwise_common.py

```diff
@@ -28,7 +28,8 @@
 
 from ._copy_utils import _empty_like_orderK, _empty_like_pair_orderK
 from ._type_utils import (
-    _acceptance_fn_default,
+    _acceptance_fn_default1,
+    _acceptance_fn_default2,
```

Comment:
Perhaps it is better to name these `_acceptance_fn_default_unary`, and `_acceptance_fn_default_binary` ?

Reply:
I agree, that's much clearer